portal-web/docroot/html/taglib/ui/input_date/page.jsp

```diff
@@ -10,7 +10,7 @@
 <%
 String randomNamespace = PortalUtil.generateRandomKey(request, "taglib_ui_input_date_page") + StringPool.UNDERLINE;
 
-if (GetterUtil.getBoolean((String)request.getAttribute("liferay-ui:input-date:disableNamespace"))) {
+if (GetterUtil.getBoolean((String)request.getAttribute("liferay-ui:input-date:disableNamespace")) || !GetterUtil.getBoolean((String)request.getAttribute("liferay-ui:input-date:useNamespace"), true)) {
 	namespace = StringPool.BLANK;
```

Comment:
@ChrisKian, the default liferay-ui:input-date:disableNamespace value is false and you also deprecated the attribute. So please remove the condition `GetterUtil.getBoolean((String)request.getAttribute("liferay-ui:input-date:disableNamespace"))`. We may directly replace it with the new attribute.